lib/src/analyzers/lint_analyzer/rules/rules_list/format_comment/format_comment_rule.dart

```diff
@@ -24,15 +24,22 @@ class FormatCommentRule extends CommonRule {
   static const _warning = 'Prefer formatting comments like sentences.';
 
   FormatCommentRule([Map<String, Object> config = const {}])
-      : super(
```

Comment:
The idea is right, but we usually create a _ConfigParser class to incapsulate the parsing. Could you create it here too? You can find an example here  https://github.com/dart-code-checker/dart-code-metrics/blob/master/lib/src/analyzers/lint_analyzer/rules/rules_list/avoid_returning_widgets/config_parser.dart

Reply:
I created a config parser here https://github.com/dart-code-checker/dart-code-metrics/pull/828/commits/06b475e10834de19ad9fed37bbafc8ee677e0d31